Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Harpy Submod Integration

So this is my first port from harpy mod and includes the basic actions you can do to not lose the game as a harpy, They are explained bellow:
 
Changelog:
- Ability to, after a siege on a province with a male thats not landed or not a council member, to round him for various actions (rape, taking prisoner, sharing with tribe, etc)
- Feeding a prisoner to your young eggs for future stat increases
- Fixing errors from harpy mating (temp)

What i wish to be able to change:
- Change how female unmarried react to a new baby being born even if without husband

Testing method:
- Play as a harpy, and siege provinces with courts in them (capitals basicaly) threw war or pillaging
- Use a prisoner to feed a baby egg in your court
- See if theres any errors or balancing issues
